docs/reference/cli/logs.md

```diff
@@ -23,7 +23,7 @@ Get logs for an app
     2020-02-05T12:47:43Z service/web/77f0e67e-4886-4aa8-be56-1d19a3aab53b ns=template id=f5b0fcdd6f63 route=root at=start method="GET" path="/" elapsed=0.038
     ....
 
-    $ convox logs --filter 2bdd60aaf431 --since 24h
+    $ convox logs --filter 2bdd60aaf431 --since 24h --no-follow
```

Comment:
I'd add just a note for the `--no-follow` option but not include in the command.